### ISSUE TITLE
Track other projects

### ISSUE THREAD

**Issue by hildjj** [peggyjs/peggy#193]
Put Plugins, generators for other languages, users of Peggy on a page in the web site.

**Comment by jonaskello** [comment]
Yes, links to eco-system packages would be helpful. It took me a while to find these two very useful packages:

- [pegjs-backtrace](https://www.npmjs.com/package/pegjs-backtrace)
- [ts-pegjs](https://www.npmjs.com/package/ts-pegjs)

**Comment by wenerme** [comment]
some grammers https://github.com/wenerme/wener/tree/master/notes/languages/parser/peg
some generator https://github.com/wenerme/wode/blob/main/packages/miniquery/src/mikro-orm/miniquery.test.ts
npm @wener/miniquery
